src/test/java/Homework16.java

```diff
@@ -0,0 +1,24 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public void Navigation(){
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        WebElement registrationLink = driver.findElement(By.cssSelector("#hel"));
+        registrationLink.click();
+        String registrationUrl= "https://bbb.testpro.io/registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(), registrationUrl);
+
```

Comment:
Please, add `driver.quit();`
The driver.quit() is used to quit the whole browser session along with all the associated browser windows, tabs and pop-ups.

For example 
![image](https://user-images.githubusercontent.com/104761482/210212203-a8aa6820-5259-474c-8320-46829d45adf3.png)